src/test/java/eu/ai4work/sws/SlidingDecisionApplicationTests.java

```diff
@@ -0,0 +1,47 @@
+package eu.ai4work.sws;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT, properties = "spring.profiles.active=logistics")
+class SlidingDecisionApplicationTests {
+
+    @Autowired
+    private TestRestTemplate testSlidingDecisionRestTemplate;
+
+    @Test
+    void contextLoads() {
+    }
+
+    @Test
+    void testLogisticsScenario() {
```

Comment:
This is just testing the "happy flow", in addition I would suggest to add several tests for different error situations, and test if the appropriate error status and message are returned. Error cases that come to mind are:

- one parameter missing
- one parameter with a typo in the name
- one additional, unknown parameter 
- JSON invalid
- one parameter with an invalid value, e.g. string instead of number

Each of those would be one test method, and each method name should make clear which case we are testing, so that the name of failed tests already gives an indication what went wrong.